Translate docs/recipes/sourcing-data

weronikadominiak opened the pull request.

weronikadominiak committed:
docs/recipes/sourcing-data translation

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -1,19 +1,20 @@
----
-title: "Recipes: Sourcing Data"
+w ---
```
Probably 'missclick'. Please, remove.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -1,19 +1,20 @@
-Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their source (e.g. the `gatsby-source-filesystem` plugin fetches data from the file system, the `gatsby-source-wordpress` plugin fetches data from the WordPress API, etc). You can also source the data yourself.
 
-## Adding data to GraphQL
+Do pozyskania danych w Gatsby używa się wtyczek, które pozyskują dane z określonych źrodeł (np. `gatsby-source-filesystem` pobiera dane z plików systemowych, `gatsby-source-wordpress` pobiera dane z WordPress Api, itp.). Możesz również pozyskać dane samodzielnie samodzielnie.
```
Please use "API" (that's the correct form, ie.: https://en.wikipedia.org/wiki/API).

mbiesiad replied:
"samodzielnie" doubled here, please remove redundant one.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -1,19 +1,20 @@
+## Dodawanie danych do GraphQL
 
-Gatsby's [GraphQL data layer](/docs/graphql-concepts/) uses nodes to model chunks of data. Gatsby source plugins add source nodes that you can query for, but you can also create source nodes yourself. To add custom data to the GraphQL data layer yourself, Gatsby provides methods you can leverage.
+[Warstwa danych GraphQL](/docs/graphql-concepts/) wykorzystuje node'y, by zmodelować porcje danych. Wtyczki źródeł dodają źródłowe node'y, które możesz pozyskać poprzez wykonanie odpowiedniego zpaytania. Możesz także stworzyć źródłowe node'y samodzielnie - Gatsby zapewnia metody, które możesz wykorzystać by dodać niestandardowe dane do warstwy danych GraphQL.
```
Typo (zpaytania). Need to be fixed, suggested:
zapytania

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -37,11 +38,11 @@ exports.sourceNodes = ({ actions, createNodeId, createContentDigest }) => {
+2. Uruchom komendę `gatsby develop`.
 
-   > _Note: After making changes in `gatsby-node.js` you need to re-run `gatsby develop` for the changes to take effect._
+   > _Uwaga: Po dokonaniu zmian w `gatsby-node.js` muisz ponownie uruchomić komendę `gatsby develop` aby zaobserwować te zmiany._
```
Typo (muisz). Please change to 'musisz'.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -37,11 +38,11 @@ exports.sourceNodes = ({ actions, createNodeId, createContentDigest }) => {
+   > _Uwaga: Po dokonaniu zmian w `gatsby-node.js` muisz ponownie uruchomić komendę `gatsby develop` aby zaobserwować te zmiany._
 
-3. Query the data (in GraphiQL or in your components).
+3. Wykonaj zapytanie (w GraphiQL lub w Twoim komponencie).
```
If I'm not mistaken, here should be 'twoim' used (written with lowercase letters). But it should be confirmed if nothing has changed.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -55,30 +56,30 @@ query MyPokemonQuery {
+- [Dokumentacja funkcji createNode](/docs/actions/#createNode)
 
-You can source Markdown data and use Gatsby's [`createPages` API](/docs/actions/#createPage) to create pages dynamically.
+## Pozyskiwanie danych z Markdownu dla blog postów i stron z GraphQL
```
"blog postów" should be changed.

mbiesiad replied:
Actually, it would be great to change a little bit, like 'wraz z GraphQL' or 'korzystając z GraphQL'.

mbiesiad reviewed (changes requested):
Please fix mentioned issues (ie. can't be typos here, in docs).

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -55,30 +56,30 @@ query MyPokemonQuery {
+## Pozyskiwanie danych z Markdownu dla blog postów i stron z GraphQL
 
-This recipe shows how to create pages from Markdown files on your local filesystem using Gatsby's GraphQL data layer.
+Możesz pozyskać dane z Markdownu i użyć [`createPages` API](/docs/actions/#createPage) by dynamicznie stworzyć strony.
```
'aby' could be better IMO, anyway, comma needed here.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -55,30 +56,30 @@ query MyPokemonQuery {
 
-This recipe shows how to create pages from Markdown files on your local filesystem using Gatsby's GraphQL data layer.
+Możesz pozyskać dane z Markdownu i użyć [`createPages` API](/docs/actions/#createPage) by dynamicznie stworzyć strony.
+Ten przepis pokazuje w jaki sposób z plikow Markdown ulokowanych na twoim lokalnym systemie stworzyć strony z użyciem warstwy danych GraphQl.
```
plikow -> plików
previously used 'Twoim', here 'twoim', must be unified.
GraphQl -> GraphQL

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -55,30 +56,30 @@ query MyPokemonQuery {
+Ten przepis pokazuje w jaki sposób z plikow Markdown ulokowanych na twoim lokalnym systemie stworzyć strony z użyciem warstwy danych GraphQl.
 
-### Prerequisites
+### Wymagania
```
Wymagania wstępne

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -55,30 +56,30 @@ query MyPokemonQuery {
+### Wskazówki
 
-1. In `gatsby-config.js`, configure `gatsby-transformer-remark` along with `gatsby-source-filesystem` to pull in Markdown files from a source folder. This would be in addition to any previous `gatsby-source-filesystem` entries, such as for images:
+1. Do pliku `gatsby-config.js`, dodaj `gatsby-transformer-remark` i `gatsby-source-filesystem` by pobrać dane z plików Markdown znajdujących się w folderze `source`. Będzie to dodatkiem do poprzednich wpisów w `gatsby-source-filesystem` jak na przykład dla obrazków:
```
'by pobrać dane' -> 'aby pobrać dane' sounds better IMO, anyway comma needed.

mbiesiad replied:
Before 'jak' also comma needed.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -94,7 +95,7 @@ module.exports = {
 ```
 
-2. Add a Markdown post to `src/content`, including frontmatter for the title, date, and path, with some initial content for the body of the post:
+2. Dodaj post z użyciem Markdownu w `src/content`, zawżyj frontmatter dla: tytułu, daty i ścieżki. Dodaj początkowy kontent do body posta:
````
Just a quick note, please be sure if we are changing the words' ​forms here (Markdownu vs Markdown, etc)

mbiesiad replied:
kontent -> zawartość
(by the way, suggested "zawżyj" -> "zawierając"; other tense etc)
'Dodaj początkowy kontent do body posta' need to be changed.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -106,7 +107,7 @@ path: /my-first-post
 ```
 
-3. Start up the development server with `gatsby develop`, navigate to the GraphiQL explorer at `http://localhost:8000/___graphql`, and write a query to get all markdown data:
+3. Uruchom serwer developerski komendą `gatsby develop`. Następnie przejdź to exploratora GraphiQL `http://localhost:8000/___graphql` i wykonaj zapytanie o dane z plikow Markdown:
````
Need to be changed - "developerski "

mbiesiad replied:
Also for change: "Następnie przejdź **to exploratora** GraphiQL"

mbiesiad replied:
Change: z **plikow** Markdown + previously used ~'Markdowna'. I'm pretty sure that should be used "Markdown" as here.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -163,7 +164,7 @@ exports.createPages = async ({ actions, graphql }) => {
 ```
 
-5. Add a post template in `src/templates`, including a GraphQL query for generating pages dynamically from Markdown content at build time:
+5. Dodaj szablon dla posta w folderze `src/templates` i umieść w nim zapytanie GraphQl by dynamicznie wygenerować strony z Markdownu w trakcie kompilacji:
````
GraphQl -> GraphQL (+ comma before 'aby/by')

mbiesiad replied:
Used different forms, once 'Markdownu', other time 'Markdown', please change.

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -198,32 +199,32 @@ export const pageQuery = graphql`
 ```
 
-6. Run `gatsby develop` to restart the development server. View your post in the browser: `http://localhost:8000/my-first-post`
+6. Wykonaj komendę `gatsby develop` by uruchomić ponownie serwer developerski. Sprawdź swój post w przeglądarce: `http://localhost:8000/my-first-post`
````
'by uruchomić' comma "," needed before aby/by

mbiesiad replied:
Change for 'developerski' needed

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -198,32 +199,32 @@ export const pageQuery = graphql`
+## Pozyskiwanie danych z Wordpress
 
-### Prerequisites
+### Wymagania
```
Wymagania wstępne

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -245,9 +246,9 @@ module.exports = {
 ```
 
-> **Note:** Refer to the [`gatsby-source-wordpress` plugin docs](/packages/gatsby-source-wordpress/?=wordpre#how-to-use) to know more about configuring your plugins.
+> **Uwaga:** Sprawdź [dokumentację wtyczki `gatsby-source-wordpress`](/packages/gatsby-source-wordpress/?=wordpre#how-to-use) by dowiedzieć się więcej o tym, jak ją prawidłowo skonfiugurować.
````
Comma ',' needed before 'by/aby'

mbiesiad replied:
Typo with 'skonfiugurować'

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
 ```
 
-5. Run `gatsby-develop` to see the newly generated pages and navigate through them.
+5. Uruchom komendę `gatsby-develop` żeby zobaczyć nowo wygenerowane strony i poruszać się pomiędzy nimi.
````
Comma ',' needed before 'by/aby/żeby'

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+5. Uruchom komendę `gatsby-develop` żeby zobaczyć nowo wygenerowane strony i poruszać się pomiędzy nimi.
 
-6. Open the `GraphiQL IDE` at `http://localhost:8000/__graphql` and open the Docs or Explorer to observe the queryable fields for `allWordpressPosts`.
+6. Otwórz`GraphiQL IDE` pod adresem `http://localhost:8000/__graphql` i otwórz Docs lub Explorer, gdzie znajdziesz pola do opytywania dla `allWordpressPosts`
```
Typo with 'opytywania'

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+6. Otwórz`GraphiQL IDE` pod adresem `http://localhost:8000/__graphql` i otwórz Docs lub Explorer, gdzie znajdziesz pola do opytywania dla `allWordpressPosts`
 
-The dynamic pages created above in `gatsby-node.js` have unique paths for navigating to particular posts, using a template component for the posts and a sample GraphQL query to source WordPress post content.
+Stworzone powyżej w `gatsby-node.js` dynamiczne strony mają unikalne ścieżki dla poszczególnych postów i używają szablonu razem z przykładowym zapytaniem GraphQl, które pozyskuje dane z Wordpressa.
```
GraphQl -> GraphQL
Wordpressa vs Wordpress (unifined, changing forms or not)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
-- More on [Sourcing from WordPress](/docs/sourcing-from-wordpress/)
-- [Live example on Sourcing from WordPress](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-wordpress)
+- [Pierwsze kroki z WordPressem i Gatsby](/blog/2019-04-26-how-to-build-a-blog-with-wordpress-and-gatsby-part-1/)
+- Więcej o [pozyskiwaniu danych z WordPressa](/docs/sourcing-from-wordpress/)
```
**Wordpressa** vs Wordpress (unifined, changing forms or not)
Below, used "danych z **WordPress**"...

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+## Pozyskiwanie danych z Contentful
 
-### Prerequisites
+### Wymagania
```
Wymagania -> requirements.
here, missing: "wstępne"

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+### Wskazówki
 
-1. Log in to Contentful with the CLI and follow the steps. It will help you create an account if you don't have one.
+1. Zaloguj się do Contenful przez CLI i postępuj zgodnie z kolejnymi krokami. Jeśli nie masz jeszcze konta, CLI umożliwi Ci założenie nowego.
```
"ci". previosuly used lower case (unify)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+2. Jeśli jeszcze nie masz żadnego space, stwórz nowy. Upewnij się, że zapisałeś ID podane pod koniec komendy. Jeśli masz już space i space ID, możesz ominąć kroki 2 i 3. 
 
-Note: for new accounts, you can overwrite the default onboarding space. Check to see the [spaces included with your account](https://app.contentful.com/account/profile/space_memberships).
+Uwaga: jeśli stworzyłeś nowe konto, możesz nadpisać domyślny space. Sprawdź [spaces zawarte w Twoim koncie](https://app.contentful.com/account/profile/space_memberships).
```
Case sensitive with 'Twoim' (previously mentioned)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -324,71 +325,69 @@ exports.createPages = async ({ graphql, actions }) => {
+Na przykład, z podmienionym ID na twoje: `contentful space seed -s '22fzx88spbp7' -t blog`
 
-4. Create a new access token for your space. Remember this token, as you will need it in step 6.
+4. Stwórz nowy klucz dostępu dla twojego space. Zapamiętaj ten klucz, będzie on potrzebny w 6tym kroku.
```
Change: 6tym

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -399,13 +398,13 @@ plugins: [
+7. Uruchom komendę `gatsby develop` i upewnij się, że zakończyła się ona sukcesem.
 
-8. Query data with the [GraphiQL editor](/docs/introducing-graphiql/) at `http://localhost:8000/___graphql`. The Contentful plugin adds several new node types to your site, including every content type in your Contentful website. Your example space with a "Blog Post" content type produces a `allContentfulBlogPost` node type in GraphQL.
+8. Wykonaj zapytanie za pomocą [edytora GraphiQL](/docs/introducing-graphiql/) pod adresem `http://localhost:8000/___graphql`. Wtyczka Contentful dodaje kilka nowych typów node'ów, włączając wszystke typy kontentu na twojej stronie Contentful. Twój przykładowy space z typem "Blog Post" tworzy node `allContentfulBlogPost` w GraphQl.
```
GraphQl to GraphQL for instance

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -399,13 +398,13 @@ plugins: [
+8. Wykonaj zapytanie za pomocą [edytora GraphiQL](/docs/introducing-graphiql/) pod adresem `http://localhost:8000/___graphql`. Wtyczka Contentful dodaje kilka nowych typów node'ów, włączając wszystke typy kontentu na twojej stronie Contentful. Twój przykładowy space z typem "Blog Post" tworzy node `allContentfulBlogPost` w GraphQl.
 
-![the graphql interface, with a sample query outlined below](../images/recipe-sourcing-contentful-graphql.png)
+![Interferjs graphQL, z przykładowym zapytaniem przedstawionym poniżej](../images/recipe-sourcing-contentful-graphql.png)
```
graphQL -> GraphQL
https://graphql.org/

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
-- [Contentful source plugin](/packages/gatsby-source-contentful/)
-- [Long-text field types returned as objects](/packages/gatsby-source-contentful/#a-note-about-longtext-fields)
-- [Example repository for this recipe](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-contentful)
+- [Tworzenie strony z użyciem Reacta o Contentful](/blog/2018-1-25-building-a-site-with-react-and-contentful/)
```
Reacta -> React
like (...) Gatsby'iego -> (...) Gatsby

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
-- [Example repository for this recipe](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-contentful)
+- [Tworzenie strony z użyciem Reacta o Contentful](/blog/2018-1-25-building-a-site-with-react-and-contentful/)
+- [Więcej o pozyskiwaniu danych z Contentful](/docs/sourcing-from-contentful/)
+- [Wtyczka źródła Contentful ](/packages/gatsby-source-contentful/)
```
"Wtyczka źródła Contentful " (redundant 'space' here, after 'Contentful').

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+- [Tworzenie strony z użyciem Reacta o Contentful](/blog/2018-1-25-building-a-site-with-react-and-contentful/)
+- [Więcej o pozyskiwaniu danych z Contentful](/docs/sourcing-from-contentful/)
+- [Wtyczka źródła Contentful ](/packages/gatsby-source-contentful/)
+- [Typy pól z długim tekstem zwracane jako obiekty](/packages/gatsby-source-contentful/#a-note-about-longtext-fields)
```
Before 'jako', please use comma ','

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+- [Przykładowe repozytorium z tym przepisem](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-contentful)
 
-## Pulling data from an external source and creating pages without GraphQL
+## Pozyskiwanie danych z zewnętrznych źródeł i tworzenie stron bez użycia GrapohQL
```
Typo: GrapohQL

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+## Pozyskiwanie danych z zewnętrznych źródeł i tworzenie stron bez użycia GrapohQL
 
-You don't have to use the GraphQL data layer to include data in pages, [though there are reasons why you should consider GraphQL](/docs/why-gatsby-uses-graphql/). You can use the node `createPages` API to pull unstructured data directly into Gatsby sites rather than through GraphQL and source plugins.
+Nie musisz używać warstwy danych GraphQl, aby uwzględnić dane na stronach, [chociaż są istotne powody, dla których powinieneś rozważyć użycie GraphQL](/docs/why-gatsby-uses-graphql/). Możesz wykorzystać `createPages` API, by pozyskać nieustrukturyzowane dane bezpośrednio do projektu Gatsby, a nie za pośrednictwem GraphQL i wtyczek źródłowych.
```
GraphQl -> GraphQL, typo

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+Nie musisz używać warstwy danych GraphQl, aby uwzględnić dane na stronach, [chociaż są istotne powody, dla których powinieneś rozważyć użycie GraphQL](/docs/why-gatsby-uses-graphql/). Możesz wykorzystać `createPages` API, by pozyskać nieustrukturyzowane dane bezpośrednio do projektu Gatsby, a nie za pośrednictwem GraphQL i wtyczek źródłowych.
 
-In this recipe, you'll create dynamic pages from data fetched from the [PokéAPI’s REST endpoints](https://www.pokeapi.co/). The [full example](https://github.com/jlengstorf/gatsby-with-unstructured-data/) can be found on GitHub.
+W tym przepisie utworzysz dynamiczne strony z danych pobranych z [endpointów PokéAPI’s REST](https://www.pokeapi.co/). [Kompletny przykład](https://github.com/jlengstorf/gatsby-with-unstructured-data/) może być znaleziony na Githubie.
```
Githubie -> GitHub

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+W tym przepisie utworzysz dynamiczne strony z danych pobranych z [endpointów PokéAPI’s REST](https://www.pokeapi.co/). [Kompletny przykład](https://github.com/jlengstorf/gatsby-with-unstructured-data/) może być znaleziony na Githubie.
 
-### Prerequisites
+### Wymagania
```
'wstępne' needed

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -456,31 +456,31 @@ export const query = graphql`
+### Wskazówki
 
-1. In `gatsby-node.js`, add the JavaScript code to fetch data from the PokéAPI and programmatically create an index page:
+1. Dodaj poniższy kod do pliku `gatsby-node.js` by pobrać dane z PokeAPI i programowo stworzyć stronę główną:
```
Comma ',' needed (by)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -506,7 +506,7 @@ exports.createPages = async ({ actions: { createPage } }) => {
 ```
 
-2. Create a template to display Pokémon on the homepage:
+2. Stwórz szablon aby wyświetlić Pokémony na stronie głównej:
````
Comma ',' needed (aby)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -526,32 +526,32 @@ export default ({ pageContext: { allPokemon } }) => (
 
-3. Run `gatsby develop` to fetch the data, build pages, and start the development server.
-4. View your homepage in a browser: `http://localhost:8000`
+3. Uruchom komendę `gatsby develop` by pozyskać dane, zbudować strony i uruchomić serwer developerski.
```
The same, comma ',' needed (by)

mbiesiad replied:
"developerski" for change

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -526,32 +526,32 @@ export default ({ pageContext: { allPokemon } }) => (
 - [Full Pokemon data repo](https://github.com/jlengstorf/gatsby-with-unstructured-data/)
-- More on using unstructured data in [Using Gatsby without GraphQL](/docs/using-gatsby-without-graphql/)
-- When and how to [query data with GraphQL](/docs/graphql-concepts/) for more complex Gatsby sites
+- Więcej o nieustrukturyzowanych danych w samouczku o [używaniu Gatsby bez GraphQl](/docs/using-gatsby-without-graphql/)
```
GraphQl -> GraphQL https://graphql.org/

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -526,32 +526,32 @@ export default ({ pageContext: { allPokemon } }) => (
+## Pozyskiwanie danych z Drupal
 
-### Prerequisites
+### Wymagania
```
'wstępne'

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -526,32 +526,32 @@ export default ({ pageContext: { allPokemon } }) => (
-- The [JSON:API module](https://www.drupal.org/project/jsonapi) installed and enabled on the Drupal site
+- [Strona Gatsby](/docs/quick-start)
+- [Drupal](http://drupal.org)
+- [JSON:API moduł](https://www.drupal.org/project/jsonapi) zainstalowany i aktywowany na Drupal
```
rearrange the words ('moduł' first)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -526,32 +526,32 @@ export default ({ pageContext: { allPokemon } }) => (
 ```
 
-2. Edit your `gatsby-config.js` file to enable the plugin and configure it.
+2. Dodaj poniższy kod do pliku `gatsby-config.js` by skonfigurować i aktywować wtyczkę.
````
Comma ',' needed before 'by/aby'

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -567,7 +567,10 @@ module.exports = {
 ```
 
-3. Start the development server with `gatsby develop`, and open the GraphiQL explorer at `http://localhost:8000/___graphql`. Under the Explorer tab, you should see new node types, such as `allBlockBlock` for Drupal blocks, and one for every content type in your Drupal site. For example, if you have a "Page" content type, it will be available as `allNodePage`. To query all "Page" nodes for their t ...
+3. Uruchom serwer developerski przy użyciu komendy `gatsby develop`, otwórz eksplorator GraphiQL pod adresem `http://localhost:8000/___graphql`. 
````
Change: developerski

mbiesiad replied:
Change: "i eden dla każdego"

mbiesiad replied:
Twoim -> twoim

mbiesiad replied:
Comma needed; jeśli

mbiesiad replied:
Typo: poniżego

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
```diff
@@ -584,9 +587,9 @@ module.exports = {
+4. By użyć swoje dane z Drupal, stwórz nową stronę `src/pages/drupal.js`, będzie ona zawierać listę wszystkich node'ów "Page" z Drupal.
 
-_**Note:** the exact GraphQL schema will depend on your how Drupal instance is structured._
+_**Uwaga:** dokładna schema GraphQL będzie zależała od tego jak skonfigurowana jest Twoja instancja Drupal._
```
twoja + comma "," (jak)

mbiesiad commented on docs/docs/recipes/sourcing-data.md:
````diff
@@ -625,10 +628,10 @@ export const query = graphql`
 ```
 
-5. With the development server running, you can view the new page by visiting `http://localhost:8000/drupal`.
+5. Mając uruchomiony serwer developerski możesz zobaczyć swoją nową stronę odwiedzając `http://localhost:8000/drupal`
````
Change: developerski

weronikadominiak committed:
recipes/sourcing-data fix typos

weronikadominiak committed:
recipes/sourcing-data fix typos

weronikadominiak commented:
@mbiesiad Thank you for the great review! Yes of course, it can't have typos! I've replaced "by" with "aby", added a few more missing commas. Changed all Wordpress and Markdown to always use original form.

mbiesiad commented:
@weronikadominiak My pleasure! Sounds awesome! 🎉 Your work was great as well! I will take a look ASAP (pm/evening). Let's keep in touch, Best wishes

weronikadominiak commented:
Closing due to https://github.com/gatsbyjs/gatsby-pl/issues/1#issuecomment-860053367
